docs/developer-docs/smart-contracts/call/overview.mdx

```diff
@@ -15,7 +15,7 @@ import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
 
 ## Introduction
 
-To interact with a <Tooltip text="A smart contract on ICP."<Tooltip> >canister's</Tooltip> methods, there are two primary types of calls that can be used: update calls and query calls.
+To interact with a <GlossaryTooltip>canister's</GlossaryTooltip> methods, there are two primary types of calls that can be used: update calls and query calls.
```

Comment:
is the text canister's (with the apostrophe) in the glossary file?

Reply:
No, it isn't. I've fixed this. Thanks!